maintainers/maintainer-list.nix

```diff
@@ -8573,6 +8573,12 @@
     githubId = 1040871;
     name = "Mathis Antony";
   };
+  svrana = {
```

Comment:
This should be in a seperate commit.

Reply:
Removed. Should I remove the maintainer line from go-minimock as well? or just keep as a string?

Reply:
Just removed it for now

Reply:
maintainer [PR](https://github.com/NixOS/nixpkgs/pull/107559) here

Reply:
> Should I remove the maintainer line from go-minimock as well?

No, please keep it.

> or just keep as a string?

That should fail ofborg.

> maintainer PR here

You can add that commit to here. The only important part is to make it a separate commit. A new PR is not necessary.

Reply:
@svrana 
your PR should have eventually two commits:

```
maintainers: add svrana
<package-name>: init at <version>
```

And in the 2nd commit put yourself in the maintainers list.

Closed https://github.com/NixOS/nixpkgs/pull/107559 for the sake of this.

Reply:
sorry for the noise. should be all fixed up now.